File: csharp/demo.officeDay/linqVsStream/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace demo.officeDay.linqVsStream
{
    public class Person:IComparable<Person>
    {

        public Person(string line)
        {
            string[] lineItems = line.Split(',');
            Name = lineItems[0];
            Country = lineItems[1];
            Age = Convert.ToInt32(lineItems[2]);
        }
        public string Name { get; set; }
        public string Country{get; set; }
        public int Age { get; set; }

        public int CompareTo(Person other)
        {
            return this.Name.CompareTo(other.Name);
        }
    }
}

File: java/src/main/java/demo/officeDay/linqVsStream/Person.java
package demo.officeDay.linqVsStream;

public class Person {
    private String name;
    private String country;
    private int age;

    public Person(String name, String country, int age){
        this.name = name;
        this.country = country;
        this.age = age;
    }

    public String getName() {
        return name;
    }

    public String getCountry() {
        return country;
    }

    public int getAge() {
        return age;
    }
}
